Submitted By: BLFS Book <blfs-book@linuxfromscratch.org>
Date: 2003-12-05
Initial Package Version: 4.44
Origin: BLFS Book
Description: Uses FHS compliant paths.

--- pine4.58/doc/pine.1.orig	2003-12-06 01:16:10.000000000 +0000
+++ pine4.58/doc/pine.1	2003-12-06 01:17:59.000000000 +0000
@@ -333,11 +333,11 @@
 .br
 /etc/mime.types	System-wide file ext. to MIME type mapping
 .br
-/usr/local/lib/pine.info	Local pointer to system administrator.
+/etc/pine.info	Local pointer to system administrator.
 .br
-/usr/local/lib/pine.conf	System-wide configuration file.
+/etc/pine.conf	System-wide configuration file.
 .br
-/usr/local/lib/pine.conf.fixed	 Non-overridable configuration file.
+/etc/pine.conf.fixed	 Non-overridable configuration file.
 .br
 /tmp/.\\usr\\spool\\mail\\xxxx	Per-folder mailbox lock files.
 .br
--- pine4.58/pine/pine.hlp.orig	2003-12-06 01:28:51.000000000 +0000
+++ pine4.58/pine/pine.hlp	2003-12-06 01:38:21.000000000 +0000
@@ -1012,9 +1012,9 @@
  executable     &lt;Unix search path&gt;/pine
  persnl cfg     ~/.pinerc
  except cfg     ~/.pinercex
- global cfg     /usr/local/lib/pine.conf
- fixed cfg      /usr/local/lib/pine.conf.fixed
- local help     /usr/local/lib/pine.info
+ global cfg     /etc/pine.conf
+ fixed cfg      /etc/pine.conf.fixed
+ local help     /etc/pine.info
 
  interrupted    ~/.pine-interrupted-mail
  debug          ~/.pine-debugN
@@ -1452,9 +1452,9 @@
 this link now to send a message to it:<BR>
 <A HREF="X-Pine-Gripe:_LOCAL_ADDRESS_?local"><!--#echo var="_LOCAL_FULLNAME_"--></A><P>
 <!--chtml endif-->
-<!--chtml if [ -r /usr/local/lib/pine.info ]-->
+<!--chtml if [ -r /etc/pine.info ]-->
 <HR WIDTH="75%">Local Support Contacts:<P>
-<!--#include file="/usr/local/lib/pine.info"-->
+<!--#include file="/etc/pine.info"-->
 <HR WIDTH="75%">
 <!--chtml endif-->
 <P> 
@@ -1542,7 +1542,7 @@
 <OL>
   <LI> <A HREF="#intro">Introduction</A>
   <LI> <A HREF="#pine_help">Pine Help</A>
-<!--chtml if [ -r /usr/local/lib/pine.info ]-->
+<!--chtml if [ -r /etc/pine.info ]-->
   <LI> <A HREF="#local_support">Local Support Contacts</A>
 <!--chtml endif-->
   <LI> <A HREF="#giving_cmds">Giving Commands in Pine</A>
@@ -1654,9 +1654,9 @@
 at your site.)
 
 
-<!--chtml if [ -r /usr/local/lib/pine.info ]-->
+<!--chtml if [ -r /etc/pine.info ]-->
 <H2><A NAME="local_support">Local Support Contacts</A></H2>
-<!--#include file="/usr/local/lib/pine.info"-->
+<!--#include file="/etc/pine.info"-->
 <!--chtml endif-->
 
 <H2><A NAME="giving_cmds">Giving Commands in Pine</A></H2>
@@ -2342,7 +2342,7 @@
 file, execute
 
 <PRE><CODE>
-		pine -conf > /usr/local/lib/pine.conf
+		pine -conf > /etc/pine.conf
 </CODE></PRE>
 <P>
 
@@ -2350,7 +2350,7 @@
 system configuration file, execute
 
 <PRE><CODE>
-		pine -P old-pine.conf -conf > /usr/local/lib/pine.conf
+		pine -P old-pine.conf -conf > /etc/pine.conf
 </CODE></PRE>
 <P>
 A system configuration file is not required.
@@ -2519,7 +2519,7 @@
 <DT> -P <EM>pinerc</EM>
 
 <DD> Uses the named file as the system wide configuration file instead of
-<EM>/usr/local/lib/pine.conf</EM> on UNIX, or nothing on <EM>PC-Pine</EM>.
+<EM>/etc/pine.conf</EM> on UNIX, or nothing on <EM>PC-Pine</EM>.
 Pinerc may be either a local file or a remote configuration folder.
 <P>
 
@@ -3247,10 +3247,10 @@
 </OL>
 <P>
 The fixed configuration file is normally
-<CODE>/usr/local/lib/pine.conf.fixed</CODE>.
+<CODE>/etc/pine.conf.fixed</CODE>.
 <P>
 The system-wide configuration file is normally
-<CODE>/usr/local/lib/pine.conf</CODE> for Unix Pine and is normally not
+<CODE>/etc/pine.conf</CODE> for Unix Pine and is normally not
 set for PC-Pine.
 For PC-Pine, if the environment variable <EM>$PINECONF</EM> is set, that
 is used for the system-wide configuration.
--- pine4.58/pine/osdep/os-lnx.h.orig	2003-12-06 01:26:39.000000000 +0000
+++ pine4.58/pine/osdep/os-lnx.h	2003-12-06 01:40:59.000000000 +0000
@@ -163,10 +163,10 @@
 
 /*----- System-wide config file ----------------------------------------*/
 #ifndef SYSTEM_PINERC
-#define SYSTEM_PINERC             "/usr/local/lib/pine.conf"
+#define SYSTEM_PINERC             "/etc/pine.conf"
 #endif
 #ifndef SYSTEM_PINERC_FIXED
-#define SYSTEM_PINERC_FIXED       "/usr/local/lib/pine.conf.fixed"
+#define SYSTEM_PINERC_FIXED       "/etc/pine.conf.fixed"
 #endif
 
 
